torchsynth/signal.py

```diff
@@ -20,3 +20,15 @@ def batch_size(self):
     def num_samples(self):
         assert self.ndim == 2
         return self.shape[1]
+
+    def new_empty(self, *args, **kwargs):
```

Comment:
Actually one thought -- the torch tensor version https://pytorch.org/docs/stable/generated/torch.Tensor.new_empty.html  has a positional arg `size` -- should this also have that as opposed to just `*args`?

Reply:
nah

*args is easier and more flexible for different torch version

Reply:
more agnostic this way

Reply:
Or does specifying it as `*args` provide a bit more flexibility and future proof wrapping this?